PurchasesCoreSwift/Logging/LoggerWrapper.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LoggerWrapper.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/08/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCLogIntent) public enum LogIntent: Int {
+    case appleError
+    case info
+    case purchase
+    case rcError
+    case rcPurchaseSuccess
+    case rcSuccess
+    case user
+    case warning
+    
+    func emojis() -> String {
+        switch self {
+        case .appleError: return Emojis.appleError.rawValue + Emojis.doubleExclamation.rawValue
+        case .info: return Emojis.info.rawValue
+        case .purchase: return Emojis.purchase.rawValue
+        case .rcError: return Emojis.rcError.rawValue + Emojis.doubleExclamation.rawValue
+        case .rcPurchaseSuccess: return Emojis.rcSuccess.rawValue + Emojis.purchase.rawValue
+        case .rcSuccess: return Emojis.rcSuccess.rawValue
+        case .user: return Emojis.appUserID.rawValue
+        case .warning: return Emojis.warning.rawValue
+        }
+    }
+}
+
+@objc(RCLoggerWrapper) public class LoggerWrapper: NSObject {
+    @objc public static func log(level: LogLevel, intent: LogIntent, message: String) {
```

Comment:
I think we could just make this a method of Logger instead of having a separate wrapper, since there aren't clear gains from having two separate public classes

Reply:
Similar to Android, I was thinking this file would be the wrapper for RCLogger. Instead of a separate log method, `appleError` could be implemented like this: 

```
@objc public static func appleError(_ message: String) {
        let messageWithPrefix = "\(LogIntent.appleError.suffix) \(message)"
        Logger.log(level: .error, message: messageWithPrefix)
    }
```
Would this implementation make more sense to justify the wrapper file?

Reply:
My thinking is that these methods might replace a lot of the stuff in Logger, which also means devs don't have to figure out which of the two to use for a given situation. 

re: mix and match of objc and non-objc stuff, I think that's fine, it's basically just a distinction between public and internal stuff. 
I would maybe separate them with pragmas or better yet extensions so that they're organized